### ISSUE TITLE
Follow #2302 refactoring removing MappingManagementService

### ISSUE THREAD

**Issue by igarashitm** [atlasmap/atlasmap#2316]
Following up #2287 - it seems CircleCI build didn't re-run after #2302 is merged into master.
```
[ERROR] /home/tomo/workspace/iPaaS/atlasmap/atlasmap/lib/itests/core/src/test/java/io/atlasmap/itests/core/MultiplicityTransformationTest.java:[289,33] cannot find symbol
[ERROR]   symbol:   variable mappingService
[ERROR]   location: class io.atlasmap.itests.core.MultiplicityTransformationTest
```
